tests/kernel/interrupt/src/nested_irq.c

```diff
@@ -55,8 +56,8 @@ void isr0(void *param)
 #ifndef NO_TRIGGER_FROM_SW
 void test_nested_isr(void)
 {
-	IRQ_CONNECT(IRQ_LINE(ISR0_OFFSET), 1, isr0, NULL, 0);
-	IRQ_CONNECT(IRQ_LINE(ISR1_OFFSET), 0, isr1, NULL, 0);
+	IRQ_CONNECT(IRQ_LINE(ISR0_OFFSET), 2, isr0, NULL, 0);
+	IRQ_CONNECT(IRQ_LINE(ISR1_OFFSET), 1, isr1, NULL, 0);
```

Comment:
FWIW: if the test is really dependent on IRQ priority being in a certain order, that deserves a comment here.  Also those priority vales are platform-dependent and probably deserve some macro hackery to make them settable per platform.

Reply:
@andyross: Thanks, I agree, provided a comment there to explain why priority is changed. The intention in the test is to get preempted by a timer while executing an ISR. So it wouldn't need a macro hackery IMHO.